terraform/services/github-actions/deploy-roles/main.tf

```diff
@@ -0,0 +1,51 @@
+data "aws_iam_policy" "developer_boundary_policy" {
+  name = "developer-boundary-policy"
+}
+
+resource "aws_iam_policy" "runner" {
+  name = "github-action-runner"
+
+  description = "Runner has access to everything within the permissions boundary to allow for terraform and other deploy steps"
+
+  policy = <<EOT
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "*",
+      "Resource": "*"
+    }
+  ]
+}
+EOT
+}
+
+resource "aws_iam_role" "github-actions-runner-role" {
+  name                 = "github-actions-runner-role"
+  permissions_boundary = data.aws_iam_policy.developer_boundary_policy.arn
+  assume_role_policy = jsonencode({
+    Statement = [
+      {
+        Action = "sts:AssumeRole"
+        Effect = "Allow"
+        Sid    = ""
+        Principal = {
+          Service = "ec2.amazonaws.com"
+        }
+      },
+      {
+        Effect = "Allow",
+        Principal = {
+          AWS = "arn:aws:iam::755619740999:role/delegatedadmin/developer/github-actions-runner-role"
```

Comment:
We should avoid including AWS account numbers in this repo. Pass the runner account number in as a terraform variable.